pkg/controllers/management/clusterupstreamrefresher/aks_upstream_spec.go

```diff
@@ -19,7 +19,6 @@ func BuildAKSUpstreamSpec(secretsCache wranglerv1.SecretCache, cluster *mgmtv3.C
 	upstreamSpec.ClusterName = cluster.Spec.AKSConfig.ClusterName
 	upstreamSpec.ResourceLocation = cluster.Spec.AKSConfig.ResourceLocation
 	upstreamSpec.ResourceGroup = cluster.Spec.AKSConfig.ResourceGroup
-	upstreamSpec.TenantID = cluster.Spec.AKSConfig.TenantID
```

Comment:
Good catch here, but it looks like this is still on the spec, we need to remove that, and anywhere else it lives: https://github.com/rancher/rancher/blob/master/pkg/client/generated/management/v3/zz_generated_aks_cluster_config_spec.go#L59

Reply:
I was waiting for `aks-operator` being merged https://github.com/rancher/aks-operator/pull/14